### ISSUE TITLE
Document package repositories in https://k8s.io/releases/download/

### ISSUE THREAD

**Issue by sftim** [kubernetes/website#43460]
**This is a Feature Request**

**What would you like to be added**
Update https://kubernetes.io/releases/download/ to mention our official package repositories

**Why is this needed**
This change will help people find official packages (`kubeadm` etc)

**Comments**
/sig docs
/sig release
/language en
/kind feature

**Comment by thisisharrsh** [comment]
I would to like to work on this issue. 
/assign

**Comment by rudrakshkarpe** [comment]
I'd like to work on this issue @sftim :)

**Comment by rudrakshkarpe** [comment]
I was wondering on whether I should include a note based on the information provided in the [official blog post](https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/#does-this-in-any-way-affect-existing-google-hosted-repositories), or if it would be more appropriate to add instructions for installing packages like ```kubeadm```, etc.  from the new repositories outlined [here](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/). 

If the approach remains completely different from what I've understood, could you please provide guidance on navigating through the official package repositories? @sftim @Pradumnasaraf

**Comment by Gauravpadam** [comment]
Here's another possibility,
We can think of a shortcode based solution here, One that renders all the official packages maintained by our teams

**Comment by professorabhay** [comment]
Hey @rudrakshkarpe, Are you still on it or should I take it and raise a PR?

**Comment by kumarankit999** [comment]
I think we can raise a PR for this!

**Comment by professorabhay** [comment]
@kumarankit999 Would you like to collaborate? 
We can work together 😉

**Comment by rudrakshkarpe** [comment]
Hi @professorabhay & @kumarankit999, If you both don't mind I'm still working on it. Possibly raise a PR by the end of the day. I welcome you both to review and help me improve as it's going to be my first contribution to CNCF :)

**Comment by sftim** [comment]
For Kubernetes SIG Docs, **anyone** can work on **any issue** (our more jargon-y way of saying this is “no [cookie licking](https://devblogs.microsoft.com/oldnewthing/20091201-00/?p=15843)”) and open a pull request to propose a related change.

**Comment by coder12git** [comment]
> @coder12git why are you taking this issue, Are you able to see the above discussions!!

Yeah i just saw that comment, but i think i can take bcoz it's been 3 weeks now :)

**Comment by coder12git** [comment]
Created a PR for this -> https://github.com/kubernetes/website/pull/44485

**Comment by vaibhav2107** [comment]
@coder12git Are you still working on this issue ?
